compiler/control/OMROptions.cpp

```diff
@@ -2215,6 +2215,19 @@ OMR::Options::jitLatePostProcess(TR::OptionSet *optionSet, void * jitConfig)
             if (_coldUpgradeSampleThreshold == TR_DEFAULT_COLD_UPGRADE_SAMPLE_THRESHOLD)
                _coldUpgradeSampleThreshold = 2;
             }
+
+         // disable DelayRelocationForAOTCompilations feature because with higher
+         // method counts, the JIT collects enough IProfiler info prior to
+         // compilation that it doesn't need to wait any longer before running the
+
+         if (self()->getOption(TR_UseHigherMethodCounts))
+            {
+            self()->setOption(TR_DisableDelayRelocationForAOTCompilations, true);// If scount has not been changed on the command line, adjust it here
+            if (self()->getInitialSCount() == TR_INITIAL_SCOUNT)
```

Comment:
Not something that has to be part of this change, but it strikes me that it would be good to have a way to determine whether a value was set explicitly or defaulted. At the moment, we'll second guess the user's `scount` if they pass `-Xjit:scount=20,useHigherMethodCounts` (with the current value of 20 for `TR_INITIAL_SCOUNT`)